test/keyring-test.js

```diff
@@ -63,4 +64,88 @@ describe('KeyRing', function() {
       uncompressed.getPublicKey(),
       KeyRing.fromMultisigScript(script, 2).getPublicKey());
   });
+
+  describe('Witness KeyRing', function() {
+    before(() => {
+      uncompressed.witness = true;
+      compressed.witness = true;
+    });
+
+    after(() => {
+      uncompressed.witness = false;
+      compressed.witness = false;
+    });
+
+    it('should get uncompressed program', () => {
```

Comment:
Test description is kinda misleading.
It's not that the program is uncompressed, it's the public key that is either compressed or uncompressed.

Maybe clarify that in test descriptions?

Reply:
You're right. All `it` messages fixed.